routers/api/v1/org/team.go

```diff
@@ -192,17 +192,28 @@ func EditTeam(ctx *context.APIContext, form api.EditTeamOption) {
 	//     "$ref": "#/responses/Team"
 
 	team := ctx.Org.Team
-	team.Description = form.Description
-	unitTypes := models.FindUnitTypes(form.Units...)
-	team.CanCreateOrgRepo = form.CanCreateOrgRepo
+	if err := team.GetUnits(); err != nil {
+		ctx.InternalServerError(err)
+	}
+
+	if form.CanCreateOrgRepo != nil {
+		team.CanCreateOrgRepo = *form.CanCreateOrgRepo
+	}
+
+	if len(form.Name) != 0 {
+		team.Name = form.Name
+	}
+
+	if len(form.Description) != 0 {
```

Comment:
So how to set the description as blank?

Reply:
@lunny good point I'll change it as i did with `CanCreateOrgRepo`